x/uibc/uics20/memo_handler.go

```diff
@@ -105,10 +112,10 @@ func (mh MemoHandler) execute(ctx *sdk.Context) error {
 	}
 
 	logger := recvPacketLogger(ctx)
-	if mh.isGMP {
-		gh := gmp.NewHandler()
-		return gh.OnRecvPacket(*ctx, mh.received, mh.memo, mh.receiver)
-	}
+	// if mh.isGMP {
```

Comment:
No need for separate execution for GMP handler because we already preprocessing the gmp handler memo on `mh.onRecvPacketPrepare`